umbraco/Umbraco.CMS.Backoffice
# fix: Issue 1170 - Markdown editor is not responsive

## Description

Fixes issue #1170 
 
## Types of changes


## Motivation and context

Fixes issue #1170 

## How to test?

> Make a Block List Editor, use the MarkDown Editor on the Element Type of the block. See this:

#1170 

## Screenshots (if appropriate)

![image](https://github.com/umbraco/Umbraco.CMS.Backoffice/assets/23620443/5d0d987d-e3ed-47c2-b117-e735442c0280)

## Checklist
